src/components/accueil/PelicoProfilNavigation.tsx

```diff
@@ -0,0 +1,93 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { Button } from '@mui/material';
+
+import { AvatarImg } from '../Avatar';
+import { CommentIcon } from '../activities/ActivityCard/CommentIcon';
+import { icons, DESC } from 'src/components/activities/utils';
+import { useActivities } from 'src/services/useActivities';
+import { primaryColor } from 'src/styles/variables.const';
+import { getUserDisplayName, toDate } from 'src/utils';
+import { ActivityType } from 'types/activity.type';
+import type { User } from 'types/user.type';
+
+export const PelicoProfilNavigation = ({ activeUser, displayAsUser = false }: { activeUser: User; displayAsUser?: boolean }) => {
+  const router = useRouter();
+  const { activities } = useActivities({
+    limit: 200,
+    page: 0,
+    type: [],
+    userId: activeUser?.id ?? 0,
+  });
+
+  const onclick = React.useCallback(() => {
+    router.push(`/activite/${activeUser.mascotteId}`);
+  }, [activeUser.mascotteId, router]);
+
+  return (
+    <>
+      <div
+        className="bg-secondary vertical-bottom-margin with-sub-header-height"
+        style={{
+          borderRadius: '10px',
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          padding: '0 0.5rem',
+        }}
+      >
+        <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', minWidth: 0 }}>
+          <span style={{ marginRight: '0.3rem', display: 'flex' }}>
+            <AvatarImg user={activeUser} size="extra-small" noLink displayAsUser={displayAsUser} onClick={onclick} style={{ cursor: 'pointer' }} />
+          </span>
+          <span className="text">
+            <strong>{getUserDisplayName(activeUser, false, displayAsUser)}</strong>
```

Comment:
Also we should write : 
` <span className="text">
            <strong>Pelico</strong>
          </span>`